# copy from OE, said to borrow from Ubuntu. Not in glib-2.0 upstream yet. Add for sanity.
#
# by Kevin Tian <kevin.tian@intel.com>, 06/25/2010
--- glib/tests/threadpool-test.c.old	2008-02-12 06:11:21.000000000 +0100
+++ glib/tests/threadpool-test.c	2008-02-12 06:11:52.000000000 +0100
@@ -5,8 +5,8 @@
 
 #include <glib.h>
 
-#define DEBUG_MSG(x)
-/* #define DEBUG_MSG(args) g_printerr args ; g_printerr ("\n");  */
+/* #define DEBUG_MSG(x) */
+#define DEBUG_MSG(args) g_printerr args ; g_printerr ("\n");
 
 #define WAIT                5    /* seconds */
 #define MAX_THREADS         10
@@ -124,10 +124,10 @@
    DEBUG_MSG (("[unused] stopping unused threads"));
    g_thread_pool_stop_unused_threads ();
 
-   DEBUG_MSG (("[unused] waiting ONE second for threads to die"));
+   DEBUG_MSG (("[unused] waiting FIVE second for threads to die"));
 
    /* Some time for threads to die. */
-   g_usleep (G_USEC_PER_SEC);
+   g_usleep (5 * G_USEC_PER_SEC);
 
    DEBUG_MSG (("[unused] stopped idle threads, %d remain, %d threads still exist",
 	       g_thread_pool_get_num_unused_threads (),
